bin/parse_phenotype.py

```diff
@@ -37,6 +37,24 @@
 assessments = ["BPQ", "TAS", "AQ", "EQ", "STAI",
     "GAD7", "MAIA", "PANAS", "PHQ9", "UCLA", "GSQ"]  # known assessments
 
+BL_one = ["participant_id", "BL_Track_meanacc", "BL_Track_RHO", "BL_Track_meanconf", "BL_Time_meanacc", "BL_Time_RHO",
+       "BL_Time_meanconf", "BL_Disc_meanacc", "BL_Disc_ROC", "BL_Disc_meanconf", "BL_d_prime", "BL_ITPE_T", "BL_ITPE_D", BL_C]
+F_one = ["participant_id", "F_Track_meanacc", "F_Track_RHO", "F_Track_meanconf", "F_Time_meanacc", "F_Time_RHO",
+       "F_Time_meanconf", "F_Disc_meanacc", "F_Disc_ROC", "F_Disc_meanconf", "F_d_prime", "F_ITPE_T", "F_ITPE_D", F_C]
+
+BL_assessment = ["participant_id", "BL_prosody_acc", "BL_face_acc", "BL_face_with_text_acc", "BL_text_acc", BL_percent_positive, BL_percent_negative,
+       "BL_percent_positive_face", "BL_percent_negative_face", "BL_FaceAndext_Acc_Positive", "BL_FaceandText_Acc_Negative",
+       "BL_Text_Acc_Positive", "BL_Text_Acc_Negative", "BL_Basic_emotions", "BL_complex_emotions",
+       "BL_Basic_face_acc", "BL_complex_face_acc", "BL_basic_FaceAndText_Acc", "BL_Complex_FaceAndText_ACC", 
+       "BL_basic_text_acc", "BL_complex_text_acc"]
+
+F_assessment = ["participant_id", "F_prosody_acc_percent", "F_face_percent", "F_face_and_text_percent", "F_text_percent", 
```

Comment:
Can we have more meaningful naming and some comments related what these variable name stands?

Reply:
This will be done once I know what the assessments are.